invokeai/frontend/web/src/features/ui/components/tabs/ModelManager/subpanels/AddModelsPanel/AdvancedAddDiffusers.tsx

```diff
@@ -92,6 +92,14 @@ export default function AdvancedAddDiffusers(props: AdvancedAddDiffusersProps) {
           label="Model Location"
           placeholder="Provide the path to a local folder where your Diffusers Model is stored"
           {...advancedAddDiffusersForm.getInputProps('path')}
+          onBlur={(e) => {
+            if (advancedAddDiffusersForm.values['model_name'] === '') {
+              advancedAddDiffusersForm.setFieldValue(
+                'model_name',
+                e.currentTarget.value.split('\\').splice(-1)[0] as string
```

Comment:
This won't work for Linux or macOS.

Also, this is unsafe... should assign the potential name to a variable and check if it is a strong first before setting.

Reply:
Why doesn't it work on Linux or macOS?

Reply:
Added check for UNIX paths.

Reply:
Actually might need to do some more stronger validation. I'll do it in a bit.

Reply:
Updated. Should now work for folders and checkpoints across both windows and unix paths. Tested on Windows. No obvious issues. Let me know if it works on Linux / macOS.